### PR TITLE
Add support for GraphQL Java 19.2

### PR DESCRIPTION
**Pull request by bclozel**
This PR adds reachability metadata for the GraphQL Java project.

Tests involve a simple query to trigger the base infrastructure.
Since GraphQL Java will perform reflection mostly on user provided types, tests also use a complete introspection query on a complex schema to trigger reflection on GraphQL Java types.

Other Frameworks like [Micronaut ship more metadata](https://github.com/micronaut-projects/micronaut-graphql/blob/master/graphql/src/main/resources/META-INF/native-image/io.micronaut.graphql/micronaut-graphql/reflect-config.json), but I didn't manage to trigger more reflection so far.

Pinging @melix as it's related to micronaut-projects/micronaut-core#7805

### CODE REVIEW

**Comment by bclozel** [comment]
@dnestoro I think this PR is ready to be considered for the next release.

**Comment by dnestoro** [comment]
> @dnestoro I think this PR is ready to be considered for the next release.

Sure It will be reviewed and merged in next few days together with other PRs for next release

File: tests/src/com.graphql-java/graphql-java/19.2/src/test/resources/graphql/greeting.graphqls [hunk 1]
```diff
@@ -0,0 +1,3 @@
+type Query {
+    greeting: String
+}
```
Comment (dnestoro) [inline tests/src/com.graphql-java/graphql-java/19.2/src/test/resources/graphql/greeting.graphqls:3]: Please add new line

File: tests/src/com.graphql-java/graphql-java/19.2/src/test/resources/graphql/starwars.graphqls [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+interface Character {
+    id: ID!
+    name: String!
+}
+
+input NewCharacter {
+    name: String
+}
```
Comment (dnestoro) [inline tests/src/com.graphql-java/graphql-java/19.2/src/test/resources/graphql/starwars.graphqls:45]: Same here

**Comment by dnestoro** [comment]
I added new lines directly from git

**Comment by dnestoro** [comment]
@bclozel please resolve conflicts

**Pull request merged**